src/Sentry/ScopeExtensions.cs

```diff
@@ -82,6 +82,15 @@ public static void AddExceptionProcessors(this Scope scope, IEnumerable<ISentryE
         public static void AddEventProcessor(this Scope scope, ISentryEventProcessor processor)
             => scope.EventProcessors.Add(processor);
 
+
+        /// <summary>
+        /// Adds expection to be ingored.
+        /// </summary>
+        /// <param name="scope">The Scope to hold the processor.</param>
+        /// <param name="ignoredTypes">The exception types to be ignored.</param>
+        public static void AddIgnoredExceptions(this Scope scope, params Type[] ignoredTypes)
+        => scope.EventProcessors.Add(new SentryEventExceptionFilterProcessor(ignoredTypes));
```

Comment:
Not sure it's the way to go to add this to the Scope.
What about this just goes into SentryOptions constructor. And passing `this._ignoredTypes` as an argument.
That means that while initializing the SDK one can call `options.AddIgnoredType<T>` which will be set to a `ConcurrentDictionary<Type, bool>` (assuming still no ConcurrentHashSet available OOTB) and inside the processor you created you can access the concurrent dictionary safely.
I hope this makes some sense

Reply:
Sorry

I don't understand what do you mean :)

Reply:
I think it's best add don't add this method here.
Adding ignored exception can only be added to `SentryOptions`. How does that sound?

Reply:
Add a method to add this on `SentryOptionsExtensions` instead